apps/web/lib/features/integrations/calendar/table-time-sheet.tsx

```
@@ -229,14 +229,17 @@ export function DataTableTimeSheet({ data }: { data?: IDailyPlan[] }) {
-                closeModal={closeModal}
-                isOpen={isOpen} />}
+            {<>
+                <RejectSelectedModal
+                    onReject={() => {
+                        // Pending implementation
+                    }}
+                    maxReasonLength={120}
+                    minReasonLength={0}
+                    closeModal={closeModal}
+                    isOpen={isOpen} />
+            </>
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)